sporkify/frontend/templates/employees.html

```diff
@@ -91,6 +92,38 @@
 		<input type="submit" id="clockInOut" name="clockInOut" value='Clock {{ clockedIn|yesno:"Out,In" }}' />
 	</form>
 </div>
+
+<div id="laborcostContainer">
+    <canvas id="doughnut-chart" width="500" height="300"></canvas>
+    <script>
+        var dv = document.getElementById("doughnut-chart")
+        var dchart = new Chart(dv, {
+            type: 'doughnut',
```

Comment:
I like doughnuts

Reply:
I LOVE DONUTSS! yeah im probably going to update the naming conventions too.